Request: add a tsconfig file for e2e specs

Angular CLI v20 will shift to a solution style tsconfig (see the changes in v20.0.0-next.6 https://github.com/cexbrayat/angular-cli-diff/compare/20.0.0-next.5...20.0.0-next.6), which means that the e2e spec files will not be included in any tsconfig of the project.

While this is not a major issue, it can be cumbersome in some scenario.

Consider the following steps:

```
# create a CLI app with solution-style config
npx @angular/cli@20.0.0-next.6 new playwright-tsconfig --defaults --no-routing
cd playwright-tsconfig

# add eslint
ng add angular-eslint@19.3.0 --skip-confirmation
# add a typed rule that necessitates the parserOptions to be set
# copy the config below
# run linter: everything is fine
ng lint

# add playwright
ng add playwright-ng-schematics@2.0.2 --install-browsers --skip-confirmation

# run linter again
ng lint

# fails with error
Linting "playwright-tsconfig"...

//playwright-tsconfig/e2e/example.spec.ts
  0:0  error  Parsing error: /playwright-tsconfig/e2e/example.spec.ts was not found by the project service. Consider either including it in the tsconfig.json or including it in allowDefaultProject

✖ 1 problem (1 error, 0 warnings)

```

eslint.config.js
```
// @ts-check
const eslint = require("@eslint/js");
const tseslint = require("typescript-eslint");
const angular = require("angular-eslint");

module.exports = tseslint.config(
  {
    files: ["**/*.ts"],
    extends: [
      eslint.configs.recommended,
      ...tseslint.configs.recommended,
      ...tseslint.configs.stylistic,
      ...angular.configs.tsRecommended,
    ],
    processor: angular.processInlineTemplates,
    rules: {
      "@angular-eslint/directive-selector": [
        "error",
        {
          type: "attribute",
          prefix: "app",
          style: "camelCase",
        },
      ],
      "@angular-eslint/component-selector": [
        "error",
        {
          type: "element",
          prefix: "app",
          style: "kebab-case",
        },
      ],
      '@angular-eslint/component-class-suffix': 'off',
      // typed rule
      '@typescript-eslint/no-deprecated': 'error'
    },
  },
  {
    files: ["**/*.html"],
    extends: [
      ...angular.configs.templateRecommended,
      ...angular.configs.templateAccessibility,
    ],
    rules: {},
  },
  {
    languageOptions: {
      parserOptions: {
        // necessary for typed rule
        projectService: true
      }
    }
  }
);
```

This is because there is no tsconfig that includes the e2e spec files.

I see two way of solving that:
- add a `tsconfig.e2e.json` file at the root of the project, and add it to the references on the main `tsconfig.json` file
- add a basic `tsconfig.json` file in the `e2e` directory that will be picked up by the tooling.

The second solution is what create-vue opted to do when both typescript and playwright are selected in a generated project, see https://github.com/vuejs/create-vue-templates/tree/main/bare-typescript-playwright-eslint/e2e

--- a/src/schematics/ng-add/files/e2e/tsconfig.json
+++ b/src/schematics/ng-add/files/e2e/tsconfig.json
@@ -1,0 +1,4 @@
+{
+  "extends": "../tsconfig.json",
+  "include": ["./**/*.ts"]
+}

--- a/src/schematics/ng-add/index.spec.ts
+++ b/src/schematics/ng-add/index.spec.ts
@@ -59,6 +59,7 @@ describe('ng-add', () => {
     const tree = await runner.runSchematic('ng-add', {}, appTree);
 
     expect(tree.files).toContain('/playwright.config.ts');
+    expect(tree.files).toContain('/e2e/tsconfig.json');
     expect(tree.files).toContain('/e2e/example.spec.ts');
 
     const packageJSON = JSON.parse(tree.readContent('/package.json'));

